megamek/src/megamek/common/cost/CostCalculator.java

```diff
@@ -42,11 +42,7 @@ public class CostCalculator {
     static double calculateCost(double[] costs) {
         double cost = 0;
         for (double v : costs) {
-            if (v < 0) {
-                cost *= -v;
-            } else {
-                cost += v;
-            }
+            cost += v;
```

Comment:
The javadoc for this method a few lines above clearly says what negative values are being used for. This change cannot be made in this form.